app/presenters/document_presenter.rb

```diff
@@ -0,0 +1,116 @@
+##
+# Base class for Blacklight document presenters
+class DocumentPresenter < Europeana::Blacklight::DocumentPresenter
+  delegate :t, to: I18n
+
+  def render_field_value(value = nil, field_config = nil)
+    safe_values = Array(value)
+
+    if field_config && field_config.limit.is_a?(Integer)
+      max_i = field_config.limit - 1
+      safe_values = safe_values[0..max_i]
+    end
+
+    safe_values.collect! { |x| x.respond_to?(:force_encoding) ? x.force_encoding("UTF-8") : x }
```

Comment:
Prefer `map!` over `collect!`.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.